chainer/dataset/tabular/slice.py

```diff
@@ -0,0 +1,143 @@
+import numbers
+import numpy as np
```

Comment:
What is the intention? Separating standard python libraries and 3rd party libraries?
If so, should I change the order of `import numpy` and `import six`?

Reply:
Yes, see this template https://docs.openstack.org/hacking/latest/user/hacking.html#import-order-template. You don't need to reorder `numpy` and `six` because both are 3rd party libraries.